pod.j2

```diff
@@ -19,17 +19,24 @@ spec:
       resources:
         requests:
           cpu: 16
+          memory: 2G
         limits:
           cpu: 16
+          memory: 2G
       command: ["cyclictest"]
       args:
-        - --smp
         - --duration={{ duration }}
         - --histofall=US
         - --mlockall
-        - --priority=80
-        - --interval=200
+        - --priority=95
+        - --interval=1000
         - --distance=0
+        - --smi
+        - -a
+        - -t 16
+        - -n
```

Comment:
`--smp`: Standard SMP testing: options -a -t -n and same priority of all threads
`-t [NUM] --threads=NUM`: number of threads: without NUM, threads = max_cpus
Therefore `--smp` == `-a -t 16 -n`

Reply:
--smp recognizes the CPU topology of the system and not of the pod. So --smp ends up running 80 threads in the 16 CPU pod. I broke it out as -a -t 16 -n to work around that.

Reply:
OK